src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -88,34 +81,38 @@ const convertInclude = async (includeBlock, ...restParams) => {
     return null;
   }
 
-  // This determines if the href has a leading '/'
-  const hrefTypePostion = () => (href.indexOf('/') === 0 ? 1 : 0);
-
-  // This checks if the supportedType is in the correct position of the href
-  const hrefIsSupported = () => supportedType =>
-    href.startsWith(supportedType, hrefTypePostion());
-
-  // This extracts the type from the href
-  const typeExtraction = Object.keys(supportedTypes).find(
-    hrefIsSupported(href),
-  );
+  const { includeType, classification } = includeClassifier({ href, pathname });
 
-  // This determines if the type is supported and returns the include type name
-  const includeType = supportedTypes[typeExtraction];
-  if (!includeType) {
+  if (
+    classification === 'not-supported' ||
+    classification === 'vj-include-not-supporting-amp'
+  ) {
     logger.info(INCLUDE_UNSUPPORTED, {
       type,
+      classification,
       url: href,
     });
     return null;
   }
 
+  let ampSrc;
+  let html;
+  if (classification === 'vj-include-supports-amp') {
+    ampSrc = ampSrcBuilder(href);
+    logger.info(INCLUDE_REQUEST_RECEIVED, {
+      url: ampSrc,
+    });
+  } else {
```

Comment:
Maybe move the line from the fetch code above (https://github.com/bbc/simorgh/pull/6873/files#diff-ace5f927a44cc0bc96a151479cba4bbaR59) to this else block:
```javascript
const url = buildIncludeUrl(href, includeType, pathname);
logger.info(INCLUDE_REQUEST_RECEIVED, {
   url,
});
html = await fetchMarkup(buildIncludeUrl(href, includeType, pathname));
```
Just keeps logs of a similar intent close by which _I think_ makes it more readable?